src/components/Header.jsx

```diff
@@ -17,6 +17,13 @@ const Header = () => {
                 </button>
               </Link>
             </li>
+            <li className="nav-item">
```

Comment:
Great job altogether. However, I think it'd be best if this dashboard icon does not show if the user is not signed in. At the moment, it always shows, even though it redirects to the login if clicked. It'd be better off not showing at all.

Reply:
On reload of the dashboard component, it redirects to the login page which is, as a result, of the isauthenticated function being false on reload. I will suggest this redirection not happen if the page is reloaded on the dashboard and user is still signed in.

Reply:
Yeah. I noticed that behaviour earlier. I'd look into it.